README.md

```diff
@@ -143,17 +247,19 @@ JSON
 }
 ```
 
-To return the `id`, `type`, `geometry`, and the Properties attribute `eo:cloud_cover`.
-This **must** return a valid STAC Item, as the includes are added to the default includes.
-Explicitly specifying `id`, `type`, and `geometry` has not effect as these are default fields,
-but `properties.eo:cloud_cover` is not a default field and thereby should be in the response.
+Return the `id`, `type`, `geometry`, and the Properties field `eo:cloud_cover`.
```

Comment:
maybe also mention here that if the geometry is excluded and you still want to return a valid STAC Item, you can set geometry to null and return the (smaller) bbox value instead.

Reply:
That _feels_ like a violation of the `exclude` semantics -- if I exclude something, that field shouldn't be present, right? Or is `null`ing something "good enough"?